src/Files.App/ViewModels/MainPageViewModel.cs

```diff
@@ -222,7 +222,8 @@ public async Task OnNavigatedToAsync(NavigationEventArgs e)
 
 			await Task.WhenAll(
```

Comment:
This is out of scope but, do you think we can move this Task.All to AppLifecycleHelper. InitializeAppComponentsAsync?

Reply:
I was a little concerned about that, but I didn't know how it would affect the behavior, so I decided to leave it as is for now.